plugins/rake-fast/rake-fast.plugin.zsh

```diff
@@ -8,7 +8,11 @@ _rake_refresh () {
 }
 
 _rake_does_task_list_need_generating () {
-  [[ ! -f .rake_tasks ]] || [[ Rakefile -nt .rake_tasks ]]
+  [[ ! -f .rake_tasks ]] || [[ Rakefile -nt .rake_tasks ]] || (_is_rails_app && [[ lib/tasks -nt .rake_tasks ]])
```

Comment:
> There is one thing I should clarify. When I do git checkout some-branch stat -f%m lib/tasks is changing. So than condition is resolved to true. I think it is how it should be. But I'd like you to confirm it.

I guess that's how it should be. Do you know of some small project that I can use to test this?